src/components/HTMLEngineProvider/index.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import withWindowDimensions from '@components/withWindowDimensions';
+import {WindowDimensionsProps} from '@components/withWindowDimensions/types';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import BaseHTMLEngineProvider from './BaseHTMLEngineProvider';
-import {defaultProps, propTypes} from './htmlEnginePropTypes';
+import HTMLEngineProviderProps from './types';
 
-function HTMLEngineProvider(props) {
+type HTMLEngineProviderWithWindowDimensionsProps = HTMLEngineProviderProps & WindowDimensionsProps;
```

Comment:
To align `index.native.tsx` and `index.ts` file props type we can switch to `useWindowDimensions` hook usage.
It will look like this
```
import React from 'react';
import * as DeviceCapabilities from '@libs/DeviceCapabilities';
import useWindowDimensions from '@hooks/useWindowDimensions';
import BaseHTMLEngineProvider from './BaseHTMLEngineProvider';
import HTMLEngineProviderProps from './types';

function HTMLEngineProvider({debug = false, children}: HTMLEngineProviderProps) {
    const {isSmallScreenWidth} = useWindowDimensions();

    return (
        <BaseHTMLEngineProvider
            debug={debug}
            textSelectable={!DeviceCapabilities.canUseTouchScreen() || isSmallScreenWidth}
        >
            {children}
        </BaseHTMLEngineProvider>
    );
}

HTMLEngineProvider.displayName = 'HTMLEngineProvider';

export default HTMLEngineProvider;
```